Advanced-Recruit-Training/README.adoc

```diff
@@ -0,0 +1,52 @@
+= U of C Solar Car Advanced Recruit Training
+Intro project to learn c++ and QT
+
+== Running the project
+
+=== Via Command Line
+
+To build this project on linux navigate to the source directory and run the following commands:
+
+`qmake`
+
+`make`
+
+`./batteryLife` - The command line arguments are as follows:
+
+[options="header"]
+|============================
+|Flag|Argument |Description
+|`-f` or `--filename` |`<filename>`|Name of csv log file of battery data
+
+|`-i` or  `--initStateOfCharge` |`<stateOfCharge>`|Initial State Of Charge of Battery as percent from 0-100
+|============================
+
+For example, to run the project opening `myData.csv` and setting the initState of charge as 50%: 
+
+`./batteryLife -f myData.csv -i 50`
+
+=== Via QT Creator
+In QT Creator, the above `qmake` and `make` are located in `Build`, and the project can be run through the play button. 
+You'll have to set the appropriate command line arguments in `Projects > Build & Run > Run`. 
```

Comment:
General in technical documents, "You" is avoided (as well as first person pronouns "I", "me", etc), it doesn't really matter for this but just so you know.